noir-projects/noir-contracts/contracts/inclusion_proofs_contract/src/main.nr

```diff
@@ -81,10 +67,12 @@ contract InclusionProofs {
         // 2) Prove the note inclusion
         if (use_block_number) {
             // docs:start:prove_note_inclusion
-            prove_note_inclusion_at(maybe_note.unwrap_unchecked(), block_number, context);
+            let header = context.get_header_at(block_number);
+            header.prove_note_inclusion(maybe_note.unwrap_unchecked());
```

Comment:
Small thing, but the `prove_note_inclusion` can be moved outside of the if/else now and reduce the constraints a lot 👀 Similar for the other tests in this one. Quite nice.